### ISSUE TITLE
exiftool error with files with long captions

### ISSUE THREAD

**Issue by garz75** [RhetTbull/osxphotos#393]
Hello, when I rune the following command: 
`osxphotos export .  --update  --convert-to-jpeg --jpeg-quality 0.95 --directory "{folder_album,NoAlbum}"  --person-keyword --keyword-template "{label}" --skip-original-if-edited --exiftool  --filename "{created.strftime,%Y-%m-%d-%H%M%S}"` 

I get weird errors with some photos which have long/unusual captions: 
```
exiftool error for file /Users/fgarzon/Desktop/out/osxphotos/2019/2019-02 Paris - Val/2019-02-03-215836.jpeg: Error: File not found - Amuse-Bouche.Error: File not found - Truite de Savoie grillée à la flamme, betteraves acidulées et Séchuan.Error: File not found - Poitrine de pintade fermière rôtie au parfum de café grillé, sarrasin et salsifis.Error: File not found - Fromages au plateau, sélection de fromages frais et affinés de SavoieError: File not found - céleri et oseille.Error: File not found - Avant dessert.Error: File not found - Effiloché d’agrumes, mousse de riz au lait, Chartreuse et baie de Timut.Error: File not found - Mignardises et chocolats.Error: File not found - Amuse-Bouche.Error: File not found - Truite de Savoie grillée à la flamme, betteraves acidulées et Séchuan.Error: File not found - Poitrine de pintade fermière rôtie au parfum de café grillé, sarrasin et salsifis.Error: File not found - Fromages au plateau, sélection de fromages frais et affinés de SavoieError: File not found - céleri et oseille.Error: File not found - Avant dessert.Error: File not found - Effiloché d’agrumes, mousse de riz au lait, Chartreuse et baie de Timut.Error: File not found - Mignardises et chocolats.Error: File not found - Amuse-Bouche.Error: File not found - Truite de Savoie grillée à la flamme, betteraves acidulées et Séchuan.Error: File not found - Poitrine de pintade fermière rôtie au parfum de café grillé, sarrasin et salsifis.Error: File not found - Fromages au plateau, sélection de fromages frais et affinés de SavoieError: File not found - céleri et oseille.Error: File not found - Avant dessert.Error: File not found - Effiloché d’agrumes, mousse de riz au lait, Chartreuse et baie de Timut.Error: File not found - Mignardises et chocolats.
Error exporting photo (A128BC7E-246C-448E-BB33-11DB56471294: IMG_0615.HEIC) as /Users/fgarzon/Desktop/out/osxphotos/2019/2019-02 Paris - Val/2019-02-03-215836.jpeg: Error: File not found - Amuse-Bouche.Error: File not found - Truite de Savoie grillée à la flamme, betteraves acidulées et Séchuan.Error: File not found - Poitrine de pintade fermière rôtie au parfum de café grillé, sarrasin et salsifis.Error: File not found - Fromages au plateau, sélection de fromages frais et affinés de SavoieError: File not found - céleri et oseille.Error: File not found - Avant dessert.Error: File not found - Effiloché d’agrumes, mousse de riz au lait, Chartreuse et baie de Timut.Error: File not found - Mignardises et chocolats.Error: File not found - Amuse-Bouche.Error: File not found - Truite de Savoie grillée à la flamme, betteraves acidulées et Séchuan.Error: File not found - Poitrine de pintade fermière rôtie au parfum de café grillé, sarrasin et salsifis.Error: File not found - Fromages au plateau, sélection de fromages frais et affinés de SavoieError: File not found - céleri et oseille.Error: File not found - Avant dessert.Error: File not found - Effiloché d’agrumes, mousse de riz au lait, Chartreuse et baie de Timut.Error: File not found - Mignardises et chocolats.Error: File not found - Amuse-Bouche.Error: File not found - Truite de Savoie grillée à la flamme, betteraves acidulées et Séchuan.Error: File not found - Poitrine de pintade fermière rôtie au parfum de café grillé, sarrasin et salsifis.Error: File not found - Fromages au plateau, sélection de fromages frais et affinés de SavoieError: File not found - céleri et oseille.Error: File not found - Avant dessert.Error: File not found - Effiloché d’agrumes, mousse de riz au lait, Chartreuse et baie de Timut.Error: File not found - Mignardises et chocolats.
```
After that, the image is exported, but the resulting file has its description truncated: 
* [IMG_0615] is the picture exported with Photos
* [2019-02-03-215836] is the version exported with osxphotos...

**Comment by garz75** [comment]
![2019-02-03-215836](https://user-images.githubusercontent.com/6645395/111035235-b7f0ea00-8419-11eb-951d-68a4245e48de.jpeg)
![IMG_0615](https://user-images.githubusercontent.com/6645395/111035240-c0e1bb80-8419-11eb-989c-0df848d580a4.jpeg)

**Comment by RhetTbull** [comment]
Sounds like a lovely menu!  :-)  I'll take a look to see if I can replicate this. I think it's likely due to embedded new lines in the caption which appear to be not properly handled.

**Comment by RhetTbull** [comment]
I think I've fixed this.  Do you mind if I include the image you posted here in the osxphotos test library?  I'll build a test case for this problem and post the fix.

**Comment by RhetTbull** [comment]
@garz75 This is fixed in [v0.41.2](https://github.com/RhetTbull/osxphotos/releases/tag/v0.41.2).  You might still get an exiftool warning for that one particular image as the caption exceeds the allowed length but exiftool will truncate it and still update the photo.  This should fix the error though.

**Comment by garz75** [comment]
> Sounds like a lovely menu! :-) I'll take a look to see if I can replicate this. I think it's likely due to embedded new lines in the caption which appear to be not properly handled.

One of the best meal of these last years... You are welcome to use the photo in your test case, no problem...

**Comment by garz75** [comment]
> @garz75 This is fixed in [v0.41.2](https://github.com/RhetTbull/osxphotos/releases/tag/v0.41.2). You might still get an exiftool warning for that one particular image as the caption exceeds the allowed length but exiftool will truncate it and still update the photo. This should fix the error though.

I confirm that the error message is gone.  Thank you!
